test/FixtureFactory.php

```diff
@@ -33,6 +33,11 @@ public static function createInvalidCard()
         return new Card('0', 12, date('Y') + 1, '123');
     }
 
+    public static function createNonChargeableCard()
+    {
+        return new Card('4000000000000002', 12, date('Y') + 1, '123');
```

Comment:
The thought occurs that we should probably create a private helper method to cut down on the duplication in the preceding three methods.